compiler/rustc_ty_utils/src/needs_drop.rs

```diff
@@ -181,12 +181,12 @@ enum DtorType {
     /// "significant" / "insignificant".
     Insignificant,
 
-    /// Type has a `Drop` implentation.
+    /// Type has a `Drop` implantation.
```

Comment:
I think this should be
```suggestion
    /// Type has a `Drop` implementation.
```